packages/kit/test/apps/basics/test/test.js

```diff
@@ -459,6 +491,23 @@ test.describe.parallel('Endpoints', () => {
 		expect(response.headers()['set-cookie']).toBeDefined();
 	});
 
+	test('HEAD with matching headers but without body', async ({ request }) => {
+		const url = '/endpoint-output/body';
+		const headResponse = await request.head(url);
```

Comment:
see above re camelCase — can we make this `responses.head` and `responses.get` etc? it's more consistent with the code style elsewhere and the extra newlines make it a bit easier to read IMHO